### ISSUE TITLE
Bcmwl can't build with kernel 5.11

### ISSUE THREAD

**Issue by kelebek333** [linuxmint/-mint20.2-beta#61]
Hi,

Uma's bcmwl-kernel-source package may upgrade from Hirsute for Edge release. http://archive.ubuntu.com/ubuntu/pool/restricted/b/bcmwl/bcmwl-kernel-source_6.30.223.271+bdcom-0ubuntu8_amd64.deb

@clefebvre 

https://forums.linuxmint.com/viewtopic.php?p=2031209#p2031209

**Comment by clefebvre** [comment]
Thanks

**Comment by clefebvre** [comment]
Confirmed, focal's version of the broadcom driver only builds for 5.4 and 5.8, despite 5.11 being currently offered as HWE-edge.

I'd like to also mention that we backported broadcom in Mint because at the time focal's version didn't support 5.8 which was used as HWE.

This really needs to be taken into account. Newly released kernel series in Ubuntu take a while to receive proper proprietary support in their package base.
